### PR TITLE
turtle skirt

### PR DESCRIPTION
**Pull request by Plyushsune**
## About the PR 

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image
image
image
image

**Changelog**

:cl:
- add: Decorated skirt
- remove: Removed fun!

### CODE REVIEW

**Comment by Plyushsune** [comment]
added reference photos to pr to convey what it looks like

**Comment by Elijahrane** [comment]
Is this a reference to something?

**Comment by Plyushsune** [comment]
No

**Comment by Elijahrane** [comment]
The sprites don't look bad but I don't get what they are supposed to be or why they are in the theatre vendor

**Comment by Vordenburg** [comment]
I like the one with the gold.

Did you have any specific plans for these? As of right now, it's hard to tell what they're for. If you just wanted to make some clothes, we could maybe find a purpose for them or give them a more generic name.

**Comment by Plyushsune** [comment]
> The sprites don't look bad but I don't get what they are supposed to be or why they are in the theatre vendor

They are for a new map, and since they are a non-nayno company, I assume the course of action is to put the uniforms in the cosplay vendor.

**Comment by Plyushsune** [comment]
> I like the one with the gold.
> 
> Did you have any specific plans for these? As of right now, it's hard to tell what they're for. If you just wanted to make some clothes, we could maybe find a purpose for them or give them a more generic name.

The same thing I told Rane, I would like to find a better name.
I assumed the general purpose would be the same thing or similar purpose to the operative uniform.

**Comment by Plyushsune** [comment]
> I like the one with the gold.
> 
> Did you have any specific plans for these? As of right now, it's hard to tell what they're for. If you just wanted to make some clothes, we could maybe find a purpose for them or give them a more generic name.

Also, that is not gold, it's a strap to latch onto the station.

**Comment by Plyushsune** [comment]
hopefully this is the last update
this pr adds once skirt to the theaterdrobe 
thats it

**Comment by Elijahrane** [comment]
you can throw it in the autodrobe probably

also the blue one with the cross looks, confusingly, like medical clothing

**Comment by Plyushsune** [comment]
> you can throw it in the autodrobe probably
> 
> also the blue one with the cross looks, confusingly, like medical clothing

i removed the blue cross because magil asked me to and it is in autodrobe

**Comment by Plyushsune** [comment]
i fixed it, so nobody will be confused by the original one

**Comment by Elijahrane** [comment]
mmm, you put it in theatre vendor. that's probably fine
we need more clothing vendors anyway now that they are getting bloated up too much

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/Uniforms/costumes.yml [hunk 1]
```diff
@@ -178,3 +178,14 @@
     sprite: Nyanotrasen/Clothing/Uniforms/Costume/arc.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Uniforms/Costume/arc.rsi
+
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformSkirtdecorskirt
+  name: Decorated turtle skirt
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Clothing/Uniforms/costumes.yml:185]: Capitalization. Should be `decorated`.

File: Resources/Prototypes/Nyanotrasen/Entities/Clothing/Uniforms/costumes.yml [hunk 2]
```diff
@@ -178,3 +178,14 @@
     sprite: Nyanotrasen/Clothing/Uniforms/Costume/arc.rsi
   - type: Clothing
     sprite: Nyanotrasen/Clothing/Uniforms/Costume/arc.rsi
+
+- type: entity
+  parent: ClothingUniformSkirtBase
+  id: ClothingUniformSkirtdecorskirt
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Clothing/Uniforms/costumes.yml:184]: Casing. A better name would be `ClothingUniformSkirtTurtle`

File: Resources/Textures/Nyanotrasen/Clothing/Uniforms/Jumpskirt/Decorskirt.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "AR or molotov",
```
Comment (Vordenburg) [inline Resources/Textures/Nyanotrasen/Clothing/Uniforms/Jumpskirt/Decorskirt.rsi/meta.json:4]: A single consistent copyright source for one author is best.

**Comment by Elijahrane** [comment]
i woke up at 2am ok, i'll just make a fix commit rq

**Comment by Plyushsune** [comment]
oh it was merged

**Pull request merged**